frontend/pages/inbox.js

```diff
@@ -96,6 +96,10 @@ export default function Inbox({ chatData, next }) {
     next: next,
   });
 
+  const resetErrorMessage = (e) => {
+    setErrorMessage(e);
+  };
+
```

Comment:
If this is generic it should be called `setErrorMessage`. If it's always clearing the message then I wouldn't allow to pass a value but always default to ""